xcp_d/interfaces/resting_state.py

```diff
@@ -100,6 +101,9 @@ class _ComputeALFFInputSpec(BaseInterfaceInputSpec):
     mask = File(exists=False,
                 mandatory=False,
                 desc=" brain mask for nifti file")
+    outputdir = File(exists=False,
+                     mandatory=False,
+                     desc="BIDS output directory")
```

Comment:
Seems like a misnomer. The variable and description indicate it's a directory, but the class is a File. I still think you should just hardcode the filename within the plotting function though.

Reply:
not sure why that was even there